enterprise_access/apps/subsidy_access_policy/models.py

```diff
@@ -375,7 +375,11 @@ def can_redeem(self, learner_id, content_key):
         Checks if the given learner_id has a number of existing subsidy transactions
         LTE to the learner enrollment cap declared by this policy.
         """
-        if len(self.transactions_for_learner(learner_id)['transactions']) < self.per_learner_enrollment_limit:
+        if self.per_learner_enrollment_limit is None:
+            return super().can_redeem(learner_id, content_key)
```

Comment:
minor nit: It's more intuitive to me if the main branch ends with `super().can_redeem()` at the bottom of the method body, and alternate branches terminate with the error response.  This is similar how we typically structure view functions, where the success case is achieved by reaching the bottom of the function, and error responses are littered in the middle.

Reply:
Refactored and will push up a new commit for re-review.